agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -133,6 +133,10 @@ type DockerClient interface {
 	// provided for the request.
 	InspectContainer(context.Context, string, time.Duration) (*types.ContainerJSON, error)
 
+	// TopContainer returns information about the top processes running in the specified container.  A timeout value and a context
+	// should be provided for the request.
+	TopContainer(context.Context, string, time.Duration) (*dockercontainer.ContainerTopOKBody, error)
```

Comment:
where is `container.ContainerTopOKBody` defined?

Reply:
https://github.com/docker/go-docker/blob/master/api/types/container/container_top.go